services/docker-compose.local.yml

```diff
@@ -45,7 +46,7 @@ services:
 
   postgres:
     ports:
-      - "5432"
+      - "5432:5432"
```

Comment:
I use a different interface to Postgres. This edit should only affect the development environment and should not influence anybody. I had to continuously change port in my configuration. That's why this was added.

Reply:
@pcrespov I think the risk is very low. Andrei uses a pycharm postgres adapter that needs the port fixed. since we now have separate docker-compose files this should be ok. I think the swarm takes care of not letting port being overriden. also the swarm uses port >=30000 for the mapping